azure-cli/graph/Throw-WhenError.ps1

```diff
@@ -0,0 +1,24 @@
+function Throw-WhenError {
```

Comment:
This function name is already used in [deploy.utilities.ps1](https://github.com/atc-net/atc-snippets/blob/1f8c1f853adec5e135b00815f681ec007fdffa7b/azure-cli/utilities/deploy.utilities.ps1), and is used extensively.
I don't know how the session will behave if both are imported, but I would imagine that it will use the last imported one ?